atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -123,11 +124,13 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji task ID: %s", self.koji_task_id)
 
-        try:
+        self.koji_build_id = self.workflow.exit_results.get(KojiImportPlugin.key)
+        if self.koji_build_id is None:
             self.koji_build_id = self.workflow.exit_results.get(KojiPromotePlugin.key)
-        except Exception:
-            self.log.exception("Failed to fetch koji build ID")
-            self.koji_build_id = None
+            if self.koji_build_id is None:
+                self.log.exception("Failed to fetch koji build ID")
```

Comment:
is that really an error at that point? You said that a null return is legitimate. I'm going with info.